linter.py

```diff
@@ -142,17 +141,24 @@ def find_errors(self, output):
                         col = pos[0]
                         end_col = pos[1]
 
-                yield lint.LintMatch(
+                match = lint.LintMatch(
                     match=error,
                     filename=file,
                     line=error['line'] - 1,
                     col=col,
                     end_col=end_col,
                     message=error_message,
                     error_type='error',
-                    code='',
+                    code=error_identifier,
                 )
 
+                processed_error = self.process_match(match, virtual_view)
+                if processed_error:
+                    # We'll display some quick actions for ignorable errors later
+                    if 'ignorable' not in error or error['ignorable']:
+                        processed_error['ignore_error'] = error['message']
```

Comment:
The conditions `'ignorable' not in error or error['ignorable']` looks redundant.

Reply:
It's mostly for backwards compatibility. If `ignorable` doesn't exist at all then directly accessing `error['ignorable']` will of course cause a `KeyError`.

Reply:
But it looks like every error is ignorable because of line 359ff.  (The blank ignore statement) Maybe you don't need `ignore_error`, on line 348 you can also do `"{msg}".format(**error)`.  ❓ 🤷

Reply:
Not every error is ignorable though, phpstan has a couple of built-in rules that can never be ignored and you can of course implement any non-ignorable custom rule. On older versions the `ignorable` property didn't yet exist (nor did identifiers), which is the reason for the blanket ignore. Newer versions require an identifier even for custom rules as far as I can tell, so it should never trigger the fallback. I can't use `{msg}` because it might also include a tip from phpstan, which can take up quite a bit of extra space and it isn't needed if you want to ignore it anyway.

I actually did realise a little problem though, I've committed another change to the new PR for that.